File: app/src/main/java/com/example/weizhenbin/wangebug/image/DefImageConfig.java
package com.example.weizhenbin.wangebug.image;

/**
 * Created by weizhenbin on 18/8/26.
 */

public class DefImageConfig {

    public static ImageConfig smallImage(){
        return new ImageConfig.Builder().setHeight(480).setHeight(480).setGif(false).build();
    }
    public static ImageConfig smallImageLong(){
        return new ImageConfig.Builder().setHeight(640).setHeight(480).setGif(false).build();
    }
}

File: app/src/main/java/com/example/weizhenbin/wangebug/image/DefImageConfig.kt
package com.example.weizhenbin.wangebug.image

/**
 * Created by weizhenbin on 18/8/26.
 */

object DefImageConfig {

    fun smallImage(): ImageConfig {
        return ImageConfig.Builder().setWidth(480).setHeight(480).setGif(false).build()
    }

    fun smallImageLong(): ImageConfig {
        return ImageConfig.Builder().setWidth(640).setHeight(480).setGif(false).build()
    }
}
